src/left-nav.jsx

```diff
@@ -218,7 +226,16 @@ let LeftNav = React.createClass({
   _getTranslateMultiplier() {
     return this.props.openRight ? 1 : -1;
   },
-
+  _enableSwipeHandlingOnClose() {
+    if (!this.props.docked && this.state.opening) {
+      document.body.addEventListener('touchstart', this._onBodyTouchStart);
```

Comment:
`addEventListener` can be replaced with a utility function in `events.js`.

```
Events.on(document.body, 'touchstart', this._onBodyTouchStart);
```
Please see:
https://github.com/callemall/material-ui/blob/master/src/utils/events.js#L15